Dockerfile

```diff
@@ -164,9 +163,11 @@ RUN /opt/ort/bin/import_proxy_certs.sh && \
             gem build cocoapods.gemspec && \
             gem install cocoapods-1.10.1.gem \
         ) && \
-        rm -rf CocoaPods-9461b346aeb8cba6df71fd4e71661688138ec21b && \
-    # Add scanners (in versions known to work).
-    curl -ksSL https://github.com/nexB/scancode-toolkit/archive/v$SCANCODE_VERSION.tar.gz | \
+        rm -rf CocoaPods-9461b346aeb8cba6df71fd4e71661688138ec21b
+
+# Add scanners (in versions known to work).
```

Comment:
Isn't the part in the parenthesis invalid?

Reply:
Well, maybe now, partly, yes. Anyway, I just moved this over. Let's eventually fix it separately.